shader/fine.wgsl

```diff
@@ -63,12 +63,13 @@ var<storage> mask_lut: array<u32, 2048u>;
 let WG_SIZE = 64u;
 var<workgroup> sh_count: array<u32, WG_SIZE>;
 
-// This is 8 winding numbers packed to a u32, 4 bits per sample
-var<workgroup> sh_winding: array<atomic<u32>, 32u>;
+// This is 4 winding numbers packed to a u32, 8 bits per sample
+var<workgroup> sh_winding: array<atomic<u32>, 64u>;
```

Comment:
As we discussed today, please document not only the packing pattern but what each shared buffer semantically represents in the hierarchy of counts.